pkg/chunk/grpc/gRPC.proto

```diff
@@ -0,0 +1,111 @@
+syntax = "proto3";
```

Comment:
I would keep consistency with filenames and rename `gRPC.proto` to the lower case `grpc.proto`. Then remember to change it in `Makefile` too.